### PR TITLE
Fixed server key case and manually setting headers

### PR DESCRIPTION
**Pull request by leocavalcante**
This PR changes the key case of the `$server` array so matches can be made.
It also manually sets the `HeaderBag` because headers are not sent as `HTTP_*` keys on Swoole's Request `$server`.

### CODE REVIEW

**Comment by susonwaiba** [comment]
Waiting for this to get merged ...

**Comment by GrahamCampbell** [comment]
@susonwaiba That comment is not helpful. :/

**Comment by susonwaiba** [comment]
@GrahamCampbell let me rephrase, tested and verified in my local so, waiting for merge. :100:
Request from swoole to symfony request mapping case issue with the route resolve. This MR fixes it.

**Comment by Nyholm** [comment]
Thank you @leocavalcante for the PR and thank you @susonwaiba for confirming that this working.

**Comment by Nyholm** [comment]
This is now released.
